docs/instruction/03-runtime.md

```diff
@@ -1,6 +1,6 @@
 # EventMesh Runtime Quick Start
 
-EventMesh Runtime is a stateful Mesh node in the EventMesh cluster, responsible for event transmission between Source Connectors and Sink Connectors. It uses Event Store as a storage queue for events.
+EventMesh Runtime is a stateful Mesh node in the EventMesh cluster, responsible for event transmission between Source Connectors and Sink Connectors. It uses EventMesh Storage as a storage queue for events.
 
 ![EventMesh Runtime](../../static/images/design-document/runtime.png)
```

Comment:
The term "Event Store" has been uniformly used in the Eventmesh-site documentation and architecture diagrams, which you can discover by reading the git history. 

The main purpose of the issue is to correct the deployment and use of the connector and storage-plugin. You can try deploying a pair of connectors yourself and fix any outdated or incorrect parts in the documentation.